### PR TITLE
feat: export externals plugin and node target plugin, electron target plugin

### PR DESCRIPTION
**Pull request by ahabhgk**
## Summary

Add export

### CODE REVIEW

File: packages/rspack/src/Compiler.ts [hunk 1]
```diff
@@ -163,12 +159,22 @@ class Compiler {
 			get EntryPlugin() {
 				return require("./builtin-plugin").EntryPlugin;
 			},
+			get ExternalsPlugin() {
+				return require("./builtin-plugin").ExternalsPlugin;
```
Comment (h-a-n-a) [inline packages/rspack/src/Compiler.ts:163]: This lazy valuation seems not working as `builtin-plugin` is just exporting everything.

Comment (ahabhgk) [reply packages/rspack/src/Compiler.ts:163]: Not affect the correctness, we can make it lazy in following PR

**Review by h-a-n-a — APPROVED** [review]

**Comment by h-a-n-a** [comment]
I think this is a `feat`.

**Comment by h-a-n-a** [comment]
@ahabhgk 
Seems it does not change the title after we adding this into a merge queue.
image

**Pull request merged**